lib/spack/docs/tutorial/examples/2.package.py

```diff
@@ -13,6 +13,8 @@ class Mpileaks(Package):
     homepage = "https://github.com/LLNL/mpileaks"
     url      = "https://github.com/LLNL/mpileaks/releases/download/v1.0/mpileaks-1.0.tar.gz"
 
+    maintainers = ['tgamblin']
```

Comment:
you could use yourself as an example :).  But I don't mind being the example either.

Reply:
If it was any other package than `mpileaks`, I would take it.